app/src/components/market/sections/market_buy/scalar_market_buy.tsx

```diff
@@ -230,7 +268,11 @@ export const ScalarMarketBuy = (props: Props) => {
   const isBuyDisabled =
     (status !== Status.Ready && status !== Status.Error) ||
     amount.isZero() ||
-    (!cpk?.isSafeApp && collateral.address !== pseudoNativeAssetAddress && hasEnoughAllowance !== Ternary.True) ||
+    Number(sharesTotal) == 0 ||
```

Comment:
I noticed this line causes this issue https://github.com/protofire/omen-exchange/issues/1722 it should be fine without it since we already have `amount.isZero()`. Same thing on `scalar_market_sell`, I think